_episodes/05-starting-jekyll.md

```diff
@@ -21,4 +21,57 @@ keypoints:
 - use it again in another page
 - change the value of the global parameter and study the effect
 
+
+> ## Creating your own variable
+>
+> Let's practice making and using variables.
+> Think of a variable you may want to use in your `index.md` page.
+> Some possible options include: your name, the institution you work at, the PI of your lab group.
+> Create the variable and reference it in your `index.md` and save/commit that to your repository.
+> What did you add to your `index.md` to create this variable?
+> Where did you add the line above to your `index.md`?
+> How did you reference that variable?
+>
+> > ## Solution
+> >
+> > We will create the `pi` variable by adding the following line to the yaml header.
+> > ~~~
+> > pi: "Ada Lovelace"
+> > ~~~
+> >
+> > Here is an example of using it in the `index.md` file
+> > ~~~
+> > This project is lead by {{ page.pi }}.
+> > ~~~
+> > {: .output}
```

Comment:
I think this and the YAML block above would be better tagged as `{: .source }`

Reply:
On reflection, I think I'm +1 to @unode's [suggestion above](https://github.com/carpentries-incubator/building-websites-with-jekyll-and-github-or-gitlab/pull/27/files#r481533910)

Reply:
The link might be broken now that I resolved the comment above.  Hopefully that means I addressed the issue but let me know if this still needs changes.